validator/accounts/v2/export.go

```diff
@@ -45,12 +46,15 @@ func ExportAccount(cliCtx *cli.Context) error {
 
 	allAccounts, err := wallet.AccountNames()
 	if err != nil {
-		log.WithError(err).Fatal("Could not get account names")
+		log.WithError(err).Error("Could not get account names")
```

Comment:
Why not return error?

And it feels out of scope of current PR which titled "Add tests... "  : )

Reply:
Oh whoops that was meant for debugging, had some weird stuff with the fatals. Thanks for pointing it out!